shared-plugins/vip-go-elasticsearch/vip-go-elasticsearch.php

```diff
@@ -126,7 +126,7 @@ public function filter__the_posts( $posts, $query ) {
 		if ( ! $query->is_main_query() || ! $query->is_search() )
 			return $posts;
 
-		if ( ! is_array( $this->search_result ) )
+		if ( is_wp_error( $this->search_result ) || ! is_array( $this->search_result ) || empty( $this->search_result['results'] ) || empty( $this->search_result['results']['hits'] ) )
```

Comment:
Would this check (`empty( $this->search_result['results']['hits'] )`) ignore the case where the search query finds `0` results?